src/components/customLoader/index.tsx

```diff
@@ -0,0 +1,2 @@
+import Loader from '@components/customLoader/Loader';
```

Comment:
same here - I have already mentioned this in the previous review, neighboring files can be imported using normal import statements